package.json

```diff
@@ -51,5 +54,8 @@
   },
   "scripts": {
     "test": "grunt ci --verbose"
+  },
+  "browser": {
+    "pouchdb/lib/deps/request": "pouchdb/lib/deps/request-browser"
```

Comment:
that’s why I said “hope” :) — we could open an issue with them to consider this as a public API.